package.json

```diff
@@ -4,6 +4,8 @@
   "repository": "lukeed/kleur",
   "description": "The fastest Node.js library for formatting terminal text with ANSI colors~!",
   "license": "MIT",
+  "main": "./index.js",
+  "types": "./index.d.ts",
```

Comment:
Let's rename the file `kleur.d.ts`. 

Then this line should be `"types": "kleur.d.ts"`.

Lastly, have to add the file to the `"files"` list, or it won't be included in npm tarball.

```json
"files": [
  "*.js",
  "*.d.ts"
],
"types": "kleur.d.ts"
```